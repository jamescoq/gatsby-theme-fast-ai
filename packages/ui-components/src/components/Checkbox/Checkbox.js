import React from 'react';

import CheckboxControl from '../CheckboxControl';
import FormField from '../FormField';

const Checkbox = props => <FormField control={CheckboxControl} kind="checkbox" {...props} />;

export default Checkbox;
