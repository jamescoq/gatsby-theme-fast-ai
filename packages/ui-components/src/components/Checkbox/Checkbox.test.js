import React from 'react';
import { shallow } from 'enzyme';

import Checkbox from './Checkbox';

describe('<Checkbox />', () => {
	const requiredProps = {};

	it('shallow', () => {
		const wrapper = shallow(<Checkbox {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
