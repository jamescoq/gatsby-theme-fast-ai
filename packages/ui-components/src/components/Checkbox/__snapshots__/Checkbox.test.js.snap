// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> shallow 1`] = `
<withGeneratedId(forwardRef(FormField))
  control={[Function]}
  kind="checkbox"
/>
`;
